src/utils/detect-platform.ts

```diff
@@ -1,6 +1,8 @@
 import path from 'path';
-import globby from 'globby';
-import { readJSONSync, existsSync, readFileSync } from 'fs-extra';
+import { globbySync } from 'globby';
+import fs from 'fs-extra';
+
+const { readJSONSync, existsSync, readFileSync } = fs;
```

Comment:
Why not do the destructuring in the `import` statement?

Reply:
Because the default `fs-extra` entry does not support esm named imports

- `fs-extra` [documentation](https://github.com/jprichardson/node-fs-extra/blob/master/README.md#esm)
- [helpful article](https://dev.to/rxliuli/re-release-fs-extra-to-properly-support-esmcjs-usage-2cim)